app/views/api/v1/orders/show.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.instance do
+  json.user @order.user_id
+  json.status @order.status
+
+  if @order.menu_items
+    json.items @order.menu_items do |item|
+      json.id item.id
+      json.item item.name
+      json.price item.price.to_f
+    end
+  else
+    json.items '"no items"'
+  end
+
+end
```

Comment:
Is there any particular reason you want to wrap that order in an instance object? I'd prefer to received something like this

```javascript
{
  'user' => user.id,
  'status' => 'pending',
  'items' => [
    {
      'id' => menu_item.id,
      'item' => menu_item.name,
      'price' => menu_item.price.to_f
    }
  ]
}
```